core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -194,7 +194,7 @@ public OutputStream getOutStream(long blockId, long blockSize, WorkerNetAddress
     }
     // No specified location to write to.
     if (address == null) {
-      throw new RuntimeException(ExceptionMessage.NO_SPACE_FOR_BLOCK_ON_WORKER.getMessage());
+      throw new RuntimeException(ExceptionMessage.NO_SPACE_FOR_BLOCK_ON_WORKER.getMessage(blockSize));
```

Comment:
It's showing the number of bytes, which is `512*1024*1024=536870912`. We have a handy method for converting bytes into a human-readable form, can you try using `FormatUtils.getSizeFromBytes`